python/batch-predictor/merlinpyspark/config.py

```diff
@@ -124,7 +124,8 @@ class MaxComputeSourceConfig(SourceConfig):
 
     def __init__(self, mc_src_proto: MaxComputeSource):
         self._proto = mc_src_proto
-        self._project, self._schema, self._table = self._proto.table.split(".")
+        self._project, self._schema, _ = self._proto.table.split(".")
+        self._table = self._proto.table
```

Comment:
Use full table name eg `<project>.<schema>.<table_name>` since `executionProject` may be different. If only `<table>` is used, it is assumed that the table is in `executionProject`, which may not always be the case.